taxi/urls.py

```diff
@@ -42,7 +47,17 @@
     path("cars/create/", CarCreateView.as_view(), name="car-create"),
     path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
+    path("cars/<int:pk>/driver_update/", car_driver_add, name="driver_update"),
+    path("cars/<int:pk>/driver_delete/", car_driver_delete,
+         name="driver_delete"),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
+    path("drivers/create/", DriverCreateView.as_view(), name="driver-create"),
+    path("drivers/<int:pk>/delete/", DriverDeleteView.as_view(),
+         name="driver-delete"),
+    path(
+        "drivers/<int:pk>/licence_update/",
+        DriverLicenseUpdateView.as_view(), name="driver-licence-update"
```

Comment:
There is a spelling inconsistency in the path 'drivers/<int:pk>/licence_update/'. The word 'licence' should be spelled 'license' to match the rest of the codebase and standard American English spelling. Update the path and corresponding view name to 'driver-license-update'.